pkg/reconciler/apis/apibinding/apibinding_reconcile_test.go

```diff
@@ -232,6 +232,11 @@ func TestReconcileBinding(t *testing.T) {
 			wantAPIExportInternalError: true,
 			wantError:                  true,
 		},
+		"APIExport doesn't have identity hash yet": {
+			apiBinding: binding.DeepCopy().
+				WithWorkspaceReference("some-workspace", "no-identity-hash").Build(),
+			wantAPIExportValid: false,
```

Comment:
just to double check: in this state the CR endpoint is not up?

Reply:
Correct. It won't create the bound CRD until there is an identity